src/views/Peers.tsx

```diff
@@ -71,22 +75,28 @@ export const Peers = () => {
     const actionsMenu = (<Menu items={itemsMenuAction} ></Menu>)
 
     const transformDataTable = (d:Peer[]):PeerDataTable[] => {
-        return d.map(p => ({ key: p.IP, ...p } as PeerDataTable))
+        const peer_ids = d.map(_p => _p.ID)
```

Comment:
Codacy has a fix for the issue: [Replace `········const·peer_ids·=·d.map(_p·=>·_p.ID)` with `····const·peer_ids·=·d.map((_p)·=>·_p.ID);`](https://app.codacy.com/gh/netbirdio/dashboard/pullRequest?prid=9560373)
```suggestion
    const peer_ids = d.map((_p) => _p.ID);
```